### ISSUE TITLE
miner kid LOL

### ISSUE THREAD

**Issue by ExoniaToken** [NighFallGT/Mercurial-Grabber#2]
![unknown](https://user-images.githubusercontent.com/95836002/149530907-1ccff3ab-6758-40e5-8547-6212df5cce14.png)
dumbass AHAHAH ADD ME ON DISCORD KID: ! Exonia#1337

**Comment by UnibIu** [comment]
yeah lol i saw that in the source code as well, i dont believe i saw anything related to malware tho; correct me if im wrong

**Comment by ingmar2** [comment]
he deleted a chat lmaoooooooooooooooooooo

**Comment by MainRobloxScripts** [comment]
how do i use this
